inference-engine/src/gna_plugin/transformations/swap_input_matmul_gna.cpp

```diff
@@ -62,11 +69,6 @@ SwapInputMatMul::SwapInputMatMul() {
             input_b_skip_fq = input_b_skip_fq->input_value(0).get_node_shared_ptr();
         }
 
-        if (!std::dynamic_pointer_cast<ngraph::opset7::Constant>(input_a_skip_fq) ||
```

Comment:
as soon as I understand input_a_skip_fq and input_b_skip_fq are not used now, consider to delete lines 61-70